ftplugin/go/splitjoin.vim

```diff
@@ -11,5 +11,4 @@ let b:splitjoin_join_callbacks = [
       \ 'sj#go#JoinVars',
       \ 'sj#go#JoinStruct',
       \ 'sj#go#JoinFunc',
-      \ 'sj#go#JoinFuncCall',
```

Comment:
That's a good point, there's a non-obvious asymmetry there. I'll add a short explanation.